test/libsolidity/smtCheckerTests/modifiers/modifier_code_after_placeholder.sol

```diff
@@ -15,6 +15,10 @@ contract C
 		assert(x > 0);
 		x = x + 1;
 	}
+
+	function g(uint _x) public {
+		x = _x;
+	}
```

Comment:
This is needed because the new checker figures that `x` can never be modified and therefore `require(x > 0);` always reverts.

Reply:
That's actually nice :-)!